threads task two.py

```diff
@@ -0,0 +1,66 @@
+"""
+Threads competition
+"""
+import time
+import threading
+
+lst_names = []
+lst_score = []
+event = threading.Event()
+
+
+def get_score(threads_name: str):
+    """
+    According to the event, the threads accessing this method try
+    to take a score from a global list and put it in another global list
+
+    :param threads_name: The name of the thread accessing this method.
+    """
+    event.wait()
+    if lst_score:
+        lst_score.pop(0)
+        lst_names.append(threads_name)
+        event.clear()
+        event.wait()
+
+
+def calculate_score():
+    """
+    Calculate which thread in a global list has the highest score and print it
+    """
+    max_wins = 0
+    max_name = ""
+    for thread_name in lst_names:
+        if lst_names.count(thread_name) > max_wins \
+                and lst_names.count(thread_name) > 1:
```

Comment:
Too much in the `if` statement. You can shorten it by exporting `lst_names.count(thread_name)` to a variable.

```suggestion
        amount = lst_names.count(thread_name)
        if amount > max_wins and amount > 1:
```